src/main/java/com/minerarcana/floralchemy/EventHandler.java

```diff
@@ -0,0 +1,16 @@
+package com.minerarcana.floralchemy;
+
+import net.minecraft.world.WorldServer;
+import net.minecraftforge.event.world.WorldEvent;
+import net.minecraftforge.fml.common.Mod.EventBusSubscriber;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@EventBusSubscriber(modid = Floralchemy.MOD_ID)
+public class EventHandler {
+    @SubscribeEvent
+    public static void onWorldLoad(WorldEvent.Load event) {
+        if(!event.getWorld().isRemote) {
```

Comment:
Casting without checking for instanceof

Reply:
!event.getWorld().isRemote already ensures that it will be a WorldServer, no? I can switch this to an instanceof check but it seems like six and two threes.

Reply:
Nope. Because sometimes stupid people (Like yours truly) do stupid stuff.